common/v2/config/constants.ts

```diff
@@ -13,3 +13,5 @@ export const DEFAULT_NETWORK_FOR_FALLBACK = 'ropsten';
 export const DEFAULT_NETWORK: NetworkId = 'Ethereum';
 
 export const DEFAULT_ASSET_DECIMAL = 18;
+
+export const MYCRYPTO_UNLOCK_CONTRACT_ADDRESS = '0x14e81196e60b128527db03d40bdba00710777805';
```

Comment:
We may need to have a static `UNLOCK_PROTOCOL_CONTRACT_ADDRESS` in the future, if we decide to do native integration of unlock token buying, so i figured `MYCRYPTO_UNLOCK` would be better, because it more describes our current contract address instance.